### ISSUE TITLE
[FEAT]: (cosmos-compose) Forward simultaneously TCP and UDP

### ISSUE THREAD

**Issue by SSTentacleSS** [azukaar/Cosmos-Server#122]
Is it possible to force cosmos-compose to forward the same port but with UDP and TCP?

```
"{Context.port}:port/tcp",
"{Context.port}:port/udp"
```

or

```
"{Context.port}:port",
"{Context.port}:port"
```

does not work

**Comment by azukaar** [comment]
Thanks!
Converted the ticket to a feat, I think there might be a tiny bit of work to do to support this

**Comment by azukaar** [comment]
fixed in 0.12
